common/include/pcl/experimental/executor/default/base_executor.hpp

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author: Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+
+#pragma once
+
+#include <pcl/experimental/executor/property.h>
+
+#include <array>
+#include <pcl/experimental/executor/trait/is_executor_available.hpp>
+#include <pcl/experimental/executor/trait/is_instance_of_base.hpp>
+#include <string>
```

Comment:
pass this through clang format to get the order right.

Reply:
There's nothing in this header file so you're including things you're not using here. Does this file need to exist?